site/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
So long as we have this configuration, we will have merge conflicts.

Although, you probably are already aware of that, and that there are some [python libraries for octokit](https://developer.github.com/v3/libraries/#python).

Reply:
I would _assume_ that a fork changes this once and forgets about it? Although _adding_ to it would be painful.